docs/tutorial/tutorial.md

```diff
@@ -191,7 +191,8 @@ Now we're passing down two props from Board to Square: `value` and `onClick`. Th
 </button>
 ```
 
-This means that when the square is clicked, it calls the onClick function that was passed by the parent. The `onClick` doesn't have any special meaning here, but it's popular to name handler props starting with `on` and their implementations with `handle`. Try clicking a square – you should get an error because we haven't defined `handleClick` yet. Add it to the Board class:
+This means that when the square is clicked, it calls the onClick function that was passed by the parent. The `onClick` here is an 
+<a href="https://facebook.github.io/react/docs/events.html" target="_blank">eventhandler</a> which handles the click event and invokes the `onClick` function received in props. Try clicking a square – you should get an error because we haven't defined `handleClick` yet. Add it to the Board class:
```

Comment:
It may be slightly confusing when we say `onClick` prop as both the Square and button has that prop. Rather rephrasing this way

The `onClick` prop of the button is part of React's synthetic event system, which will call the function passed in when a click event is dispatched on the button.